crossplane/ext/abstract.py

```diff
@@ -7,8 +7,9 @@
 
 class CrossplaneExtension(object):
 
-    def __init__(self):
-        self.directives = {}
+    @abc.abstractproperty
+    def directives(self):
+        pass
```

Comment:
`abc.abstractproperty` does literally nothing unless the class lists `abc.ABCMeta` as a metaclass.

just do
```python
class CrossplaneExtension(object):
    directives = {}
```